core/src/main/java/org/mule/runtime/core/internal/metadata/MuleMetadataManager.java

```diff
@@ -8,11 +8,12 @@
 
 import static java.lang.String.format;
 import static org.mule.runtime.core.config.i18n.MessageFactory.createStaticMessage;
-
 import org.mule.runtime.api.metadata.ComponentId;
+import org.mule.runtime.api.metadata.MetadataKeysAware;
```

Comment:
rename package to datasense... Change all the occurences of Metadata to DataSense